sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -752,6 +871,12 @@ public void testInferKeyCoder() {
             instanceof VarLongCoder);
   }
 
+  @Test(expected = RuntimeException.class)
```

Comment:
It's better to use `@Rule ExpectedException` because it allows you to specify the expectations on the message etc., and is generally widely used in Beam